Lib/ast.py

```diff
@@ -213,6 +226,77 @@ def get_docstring(node, clean=True):
     return text
 
 
+def _splitlines_no_ff(source):
```

Comment:
Another option is to use `.splitlines(keepends=True)` (which considers `\f` as an endline) and then re-join those that aren't ending in `\r` or `\n`, I am not sure which approach is better.